<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数字炸弹</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/pic/images/RY.ico</normaloff>:/pic/images/RY.ico</iconset>
  </property>
  <property name="statusTip">
   <string>就绪</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="statusTip">
       <string>就绪</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label1">
       <property name="font">
        <font>
         <family>华文新魏</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="statusTip">
        <string>就绪</string>
       </property>
       <property name="text">
        <string>请输入数字：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label2">
       <property name="font">
        <font>
         <family>华文新魏</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="statusTip">
        <string>就绪</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>64</width>
          <height>48</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="inlineEdit">
          <property name="statusTip">
           <string>输入栏</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="okbutton">
            <property name="font">
             <font>
              <family>等线</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="statusTip">
             <string>确认</string>
            </property>
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="flushbutton">
            <property name="font">
             <font>
              <family>等线</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="statusTip">
             <string>刷新</string>
            </property>
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="endbutton">
            <property name="font">
             <font>
              <family>等线</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="statusTip">
             <string>退出</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>64</width>
          <height>48</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>423</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="statusTip">
     <string>菜单栏</string>
    </property>
    <property name="title">
     <string>模式选择</string>
    </property>
    <addaction name="updatenum"/>
    <addaction name="separator"/>
    <addaction name="updatepattaern"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string>状态栏</string>
   </property>
  </widget>
  <action name="initpattern">
   <property name="icon">
    <iconset>
     <normaloff>../images/Water Elemental.ico</normaloff>../images/Water Elemental.ico</iconset>
   </property>
   <property name="text">
    <string>初始模式</string>
   </property>
  </action>
  <action name="updatepattaern">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/pic/images/Fire Elemental.ico</normaloff>:/pic/images/Fire Elemental.ico</iconset>
   </property>
   <property name="text">
    <string>更新模式</string>
   </property>
   <property name="statusTip">
    <string>每局重新更新一次范围</string>
   </property>
  </action>
  <action name="updatenum">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/pic/images/doc.ico</normaloff>:/pic/images/doc.ico</iconset>
   </property>
   <property name="text">
    <string>更新范围</string>
   </property>
   <property name="statusTip">
    <string>重新输入范围</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
